reports/helmcharts/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source set_env.sh
+python helm-chart-versions.sh
```

Comment:
Its not a python script so you wont use `python <scriptname` here, you'll just use normal bash options
`./helm-chart-versions.sh`